service/payment.go

```diff
@@ -39,7 +39,14 @@ func (service *PaymentService) CreatePaymentSession(w http.ResponseWriter, req *
 		return
 	}
 
-	paymentResource, err := getPaymentResource(w, req, incomingPaymentResourceRequest.Resource)
+	cfg, err := config.Get()
```

Comment:
we should get the main config (reading in env vars/flags etc) once on application startup rather than doing it on each request